observability/index.adoc

```diff
@@ -23,9 +23,13 @@ ifndef::openshift-dedicated,openshift-rosa[]
 * {PM-shortname-c}
 endif::openshift-dedicated,openshift-rosa[]
 
-
 {ObservabilityLongName} connects open-source observability tools and technologies to create a unified {ObservabilityShortName} solution. The components of {ObservabilityLongName} work together to help you collect, store, deliver, analyze, and visualize data.
 
+[NOTE]
+====
+With the exception of monitoring, {ObservabilityLongName} components have distinct release cycles separate from the core {product-title} release cycles. See the Red{nbsp}Hat link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] page for their release compatibility.
+====
+
```

Comment:
L33 Please refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-subsections-level-1-and-lower-headings

For any Level 1 or lower subsections in an assembly (for example, ==, ===, etc.), add a _{context} variable to the end of anchor IDs

Reply:
Thank you for noticing! I added the variable :+1: